src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Main {
+
+    public static void main(String[] args) {
+
```

Comment:
better don't leave an empty line after method signature. You can write the code on the next line after the method's signature.

Reply:
done.